test/integration/org/controls/gcp.rb

```diff
@@ -0,0 +1,357 @@
+  end
+
+  describe google_project(project: org_audit_logs_project_id) do
+    it { should exist }
+    its('project_id') { should cmp org_audit_logs_project_id }
+    its('lifecycle_state') { should cmp 'ACTIVE' }
+  end
+
+  describe google_project(project: org_billing_logs_project_id) do
+    it { should exist }
+    its('project_id') { should cmp org_billing_logs_project_id }
+    its('lifecycle_state') { should cmp 'ACTIVE' }
+  end
+
+  describe google_project(project: org_secrets_project_id) do
+    it { should exist }
+    its('project_id') { should cmp org_secrets_project_id }
+    its('lifecycle_state') { should cmp 'ACTIVE' }
+  end
+
+  describe google_project(project: interconnect_project_id) do
+    it { should exist }
+    its('project_id') { should cmp interconnect_project_id }
+    its('lifecycle_state') { should cmp 'ACTIVE' }
+  end
+
+  describe google_project(project: dns_hub_project_id) do
+    it { should exist }
+    its('project_id') { should cmp dns_hub_project_id }
+    its('lifecycle_state') { should cmp 'ACTIVE' }
+  end
+
+  describe google_project(project: scc_notifications_project_id) do
+    it { should exist }
+    its('project_id') { should cmp scc_notifications_project_id }
+    its('lifecycle_state') { should cmp 'ACTIVE' }
+  end
+
+  logs_apis.each do |api|
+    describe google_project_service(
+      project: org_audit_logs_project_id,
+      name: api
+    ) do
+      it { should exist }
+      its('state') { should cmp 'ENABLED' }
+    end
+  end
+
+  logs_apis.each do |api|
+    describe google_project_service(
+      project: org_billing_logs_project_id,
+      name: api
+    ) do
+      it { should exist }
+      its('state') { should cmp 'ENABLED' }
+    end
+  end
+
+  org_secrets_apis.each do |api|
+    describe google_project_service(
+      project: org_secrets_project_id,
+      name: api
+    ) do
+      it { should exist }
+      its('state') { should cmp 'ENABLED' }
+    end
+  end
+
+  scc_notifications_apis.each do |api|
+    describe google_project_service(
+      project: scc_notifications_project_id,
+      name: api
+    ) do
+      it { should exist }
+      its('state') { should cmp 'ENABLED' }
+    end
+  end
+
+  dns_hub_apis.each do |api|
+    describe google_project_service(
+      project: dns_hub_project_id,
+      name: api
+    ) do
+      it { should exist }
+      its('state') { should cmp 'ENABLED' }
+    end
+  end
+
+  describe google_pubsub_topic(
+    project: scc_notifications_project_id,
+    name: 'top-scc-notification'
+  ) do
+    it { should exist }
+  end
+
+  describe google_pubsub_subscription(
+    project: scc_notifications_project_id,
+    name: 'sub-scc-notification'
+  ) do
+    it { should exist }
+  end
+
+  bigquery_logs_datasets.each do |dataset|
+    describe google_bigquery_dataset(
+      project: org_audit_logs_project_id,
+      name: dataset
+    ) do
+      it { should exist }
+    end
+  end
+
+  if parent_resource_type == 'folder'
```

Comment:
As mentioned, if we know that tests are running inside a folder we might not need this

Reply:
thanks. now the tests assume they run under a folder.